ddp-workspace/projects/ddp-atcp/src/app/components/activityForm/app-atcp-activity.component.ts

```diff
@@ -31,7 +32,7 @@ import * as Routes from '../../router-resources';
   <main class="main main_activity" [ngClass]="{'main_sticky': isLoaded && model && model.subtitle, 'feeding-survey': model?.activityCode === ActivityCodes.FEEDING}">
         <ng-container *ngIf="isLoaded && model">
             <section class="section">
-                <ddp-subject-panel></ddp-subject-panel>
+                <ddp-subject-panel [invitationId]="invitationId"></ddp-subject-panel>
```

Comment:
@MocanaAtBroad do we need this panel here? I'm not sure there is an admin account for atcp. If we plan to add it in the future I'd keep it otherwise I'd prefer to remove it to simplify the logic a little bit.